lib/tls/certificate.toit

```diff
@@ -102,3 +102,17 @@ Returns the hash of the added certificate, which can be used to add it more
 */
 add-global-root-certificate_ cert hash/int?=null -> int:
   #primitive.tls.add-global-root-certificate
+
+/**
+Adds the trusted root certificates that are installed on the system.
+
+This is only supported on Windows.  On other platforms it currently does
+  nothing.
```

Comment:
```suggestion
This is only supported on Windows.  On other platforms it does nothing.
```

Reply:
We might add this functionality on MacOS too, so I think the original is better.

Reply:
We can then update the comment.